app/reducers/channels.js

```diff
@@ -657,6 +662,15 @@ channelsSelectors.closingChannelIds = createSelector(
   ]
 )
 
+channelsSelectors.closedChannels = createSelector(
+  closedChannelsSelector,
+  nodesSelector,
+  closingChannelIdsSelector,
+  (closedChannels, nodes, closingChannelIds) => [
+    ...closedChannels.map(channelObj => decorateChannel(channelObj, nodes, closingChannelIds))
```

Comment:
Is there a need to do a spread since `map` already creates a new array?

Reply:
Same - updated here.